blog/_posts/2018-04-18-scala-3.md

```diff
@@ -0,0 +1,104 @@
+---
+layout: blog-detail
+post-type: blog
+by: Martin Odersky
+title: "Towards Scala 3"
+---
+
+Now that Scala 2.13 is around the corner, it's time to consider the
```

Comment:
I think "around the corner" is not the right wording. There is still M4, then M5, then RC1. I would not consider 2.13 "around the corner" until RC1 is out. I think you might simply omit the whole sentence, and then in the "When will it come out?" section, insert "Scala 2.13 will come out later this year."